### PR TITLE
Fix and issue related to not picking the local rustfmt.toml

### PR DESCRIPTION
**Pull request by chevdor**
See https://github.com/statiolake/vscode-rustfmt/issues/1

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -11,9 +11,13 @@ function rangeWholeFile(doc: vscode.TextDocument): vscode.Range {
 }
 
 function getFormattedString(doc: vscode.TextDocument): string {
+	const documentUri = vscode.Uri.parse(doc.fileName);
```
Comment (statiolake) [inline src/extension.ts]: Perhaps we should use `doc.uri` directly instead. I couldn't parse the file name as a URI correctly on Windows (drive letters like "C:" was treated as a scheme).

**Review by statiolake — CHANGES_REQUESTED** [review]
Thanks for the PR! I did not notice the issue until now.

I updated the dependencies before merging this PR because I couldn't even test the extension with too old `vscode` package... It seemed type definition `vscode.d.ts` was no longer available for the old package, so I replaced to `@types/vscode` package. I'm sorry for the merge conflicts in package.json.

File: src/extension.ts (hunk 1 above)
Comment (chevdor) [reply src/extension.ts]: fixed

File: package.json [hunk 2]
```diff
@@ -22,6 +22,9 @@
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./",
 		"watch": "tsc -watch -p ./",
+		"postinstall": "node ./node_modules/vscode/bin/install",
+		"test": "npm run compile && node ./node_modules/vscode/bin/test",
```
Comment (statiolake) [inline package.json]: Those two lines ("postinstall" and "test") are no longer needed. Could you remove them?

**Review by statiolake — CHANGES_REQUESTED** [review]
Sorry to bother you again. Once those outdated lines are removed, I'll squash merge this. Thanks!

File: package.json (hunk 2 above)
Comment (chevdor) [reply package.json]: Well, it sounds like it goes beyond the scope of my PR :)
Feel free to commit those changes directly but you probably should do another PR or do it directly in master as this is absolutely not related to the issue the PR is fixing.

Comment (statiolake) [reply package.json]: They were actually removed in #3, the PR I merged after you create this PR. However those lines are going to be reintroduced by this PR. It's probably accidentaly included when resolving merge conflicts. Sorry for confusion.

Comment (chevdor) [reply package.json]: Merging #2 before #3 would have been a good idea 😄
Done

**Review by statiolake — APPROVED** [review]
Looks good now. Thanks!

**Pull request merged**